digital-deck-ui/src/components/Game.vue

```diff
@@ -11,7 +11,6 @@
 import HandArea from './HandArea.vue';
 import TableArea from './TableArea.vue';
 import OtherPlayersArea from './OtherPlayersArea.vue';
-import { VueCookies } from "vue-cookies";
```

Comment:
Do cookies work now? I remember when I added cookies there has to be this import statement even though it wasn't used. Can you verify?

Reply:
Oh cookies isn't used in this component so we don't need it